scripts/master.sh

```diff
@@ -26,7 +26,8 @@ ansible -i /dev/null \
 		detach=true \
 		command='/usr/lib/systemd/systemd \
 		--system' \
-		privileged=true"
+        capabilities=SYS_ADMIN \
```

Comment:
Check the white space indenting here. These lines don't match the ones above them.